### DISCUSSION TITLE
Welche Rebase-Strategie sollte man anwenden?

### DISCUSSION THREAD

**Discussion by tgrothe** [Programmiermethoden/Dungeon#742] [Announcements]
In #680 ist beim Rebasen etwas verloren gegangen, ich hatte diese Situation aber auch schon.

Welche Strategie sollte beim Rebasen verfolgt werden?

Ich mache meist `git rebase -X ours master` und schaue dann, welche Änderungen noch nicht da sind...

**Comment by cagix** [comment]
Ich glaube, im konkreten Fall ist es einfach nur untergegangen, weil es parallel wirklich viele Änderungen gab. Da wird das einfach schnell ziemlich unübersichtlich. Die einzige Abhilfe wäre, mit kleineren PR's zu arbeiten und größere Strukturänderungen nicht parallel zu machen (nur manchmal geht das nicht).

**Reply by tgrothe** [reply]
ja, ok, danke.
